test/cmd/admin.sh

```diff
@@ -107,7 +108,11 @@ echo "certs: ok"
 os::test::junit::declare_suite_end
 
 os::test::junit::declare_suite_start "cmd/admin/groups"
-os::cmd::expect_success 'oadm groups new group1 foo bar'
+os::cmd::expect_success_and_text 'oadm groups new shortoutputgroup -o name' 'group/shortoutputgroup'
+os::cmd::expect_failure_and_text 'oadm groups new shortoutputgroup' 'Error from server: groups "shortoutputgroup" already exists'
+os::cmd::expect_failure_and_text 'oadm groups new errorgroup -o json' 'error: Unexpected -o output mode: json'
```

Comment:
> I'm not sure these new test cases give us enough new coverage to be worth it since the printer is pretty ubiquitous, but they're quick tests so I'm inclined to keep them. I'll defer to @deads2k

these are fine.